crates/jwt/src/jwe.rs

```diff
@@ -0,0 +1 @@
+// implementation for JWT's as JWE's
```

Comment:
torn on if we should leave this or remove it, I'm leaning towards remove it

Reply:
decided to stub in scaffolded code, but not implement and instead created ticket for subsequent work #174